service/create/launch_configuration.go

```diff
@@ -91,21 +93,23 @@ func (s *Service) createLaunchConfiguration(input launchConfigurationInput) (boo
 		return false, microerror.MaskAny(err)
 	}
 
-	launchConfigName, err := launchConfigurationName(input.cluster, input.prefix)
+	launchConfigName, err := launchConfigurationName(input.cluster, input.prefix, securityGroupID)
```

Comment:
It worries me that the SG IDs are changing. Any idea what caused that?

Although I think we'll need to update the launch config as part of updating worker nodes. So if this makes the launch config more robust maybe it's a good thing.